subprojects/configuration-cache/src/main/kotlin/org/gradle/configurationcache/ConfigurationCacheException.kt

```diff
@@ -42,7 +42,7 @@ class ConfigurationCacheError internal constructor(
 
 
 @Contextual
-sealed class ConfigurationCacheException private constructor(
```

Comment:
Let's keep it `protected` then.

Reply:
So because it's `sealed class` so protected is not necessary?

![image](https://user-images.githubusercontent.com/12689835/155046888-ef17b48e-131e-4ee3-96d9-94e7b47b07e9.png)

Reply:
I see 👍